### ISSUE TITLE
Allow external methods in goverter:mapExtend

### ISSUE THREAD

**Issue by shuqingzai** [jmattheis/goverter#47]
**Have you read the project readme?**

- [x] Yes, but it does not include related information regarding my question.
- [x] Yes, but the steps described do not work.
- [x] Yes, but I am having difficulty understanding it and want clarification.

**Describe your question**
A clear and concise description of what the question is. Include errors and go source files.

**I tried importing packages, splitting packages and functions with `:` or `.` to no avail**

```go
// goverter:converter
type Converter interface {
    // goverter:mapExtend FullName github.com/jmattheis/goverter.ExtendFullName
    // goverter:mapExtend Age github.com/jmattheis/goverter:ExtendFullAge
    Convert(source Input) Output
}
```

**Comment by jmattheis** [comment]
Yeah, this is currently unsupported, but it sounds like a useful addition.

**Comment by jmattheis** [comment]
Fixed in [v0.13.0](https://github.com/jmattheis/goverter/releases/tag/v0.13.0)
